.github/workflows/ci_functests-install.yml

```diff
@@ -2,9 +2,13 @@ name: Hub-CI
 
 on:
   push:
-    branches: [ wip_lapi_xxx_no ]
+    branches:
+      - wip_lapi
+      - master
   pull_request:
-    branches: [ wip_lapi_xxx_no ]
+    branches:
+      - wip_lapi
+      - master
```

Comment:
This'll have to be changed in the final release.

Reply:
I guess we can just remove `wip_lapi` ? or you see something else?

Reply:
fair enough